releasenotes/notes/remove-pilot-tracing.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+  - |
+    **Removed** pilot flag `PILOT_TRACE_SAMPLING`.
```

Comment:
'Instead users should use ????' ( I assume OTEL recommended env variables - but users shouldn't guess). A link to some doc (or wiki page or something ) would be nice too.

Reply:
update upgrade notes